### PR TITLE
Changelog for the release 2.45.0

### PR DESCRIPTION
**Pull request by numanelahi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# Release 2.45.0 13-07-2023
+
+- Update Go version to 1.20 (#831)
+- Add llvm dependency for scudo allocator (#834)
```
Comment (nareddyt) [inline CHANGELOG.md:4]: Not needed, not customer visible

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+- fix interface for listener creation (#828)
```
Comment (nareddyt) [inline CHANGELOG.md:5]: Not needed

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+- Update base alpine image to 3.18 (#827)
+- expose --health_check_grpc_backend_no_traffic_interval (#815)
+- b/277969940: Upgrade config generator libraries to use golang proto v2 (#814)
```
Comment (nareddyt) [inline CHANGELOG.md:8]: Not needed

Comment (numanelahi) [reply CHANGELOG.md]: Removed the unnecessary changes

**Pull request merged**